src/views/tickets/view-link.php

```diff
@@ -63,5 +63,10 @@
 
 <div class="tribe-link-view-attendee">
 	<?php echo $message ?>
-	<a href="<?php echo esc_url( $link ) ?>"><?php echo sprintf( esc_html__( 'View your %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, $user_id ) ) ?></a>
-</div>
\ No newline at end of file
+	<a href="<?php echo esc_url( $link ); ?>">
+		<?php
+		// Translators: %s: The name(s) of the type(s) of ticket(s) the specified user (optional) has for the specified event.
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$view'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$user_id'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$user_id'.

<sub>hash: fda861ac30e3acbf0d8df40207fb052f</sub>

Reply:
The arguments in next line are ultimately escaped already.